db/db/src/main/java/io/helidon/db/DbExecute.java

```diff
@@ -0,0 +1,407 @@
+     */
+    default CompletionStage<Long> namedUpdate(String statementName, Object... parameters) {
+        return createNamedUpdate(statementName).params(parameters).execute();
+    }
+
+    /**
+     * Create and execute update statement using a statement passed as an argument.
+     *
+     * @param statement  the update statement to be executed
+     * @param parameters query parameters to set
+     * @return number of rows updateed into the database
+     */
+    default CompletionStage<Long> update(String statement, Object... parameters) {
+        return createUpdate(statement).params(parameters).execute();
+    }
+
+    /*
+     * DELETE
+     */
+
+    /**
+     * Create a delete statement using a named statement passed as an argument.
+     *
+     * @param statementName the name of the statement
+     * @param statement the statement text
+     * @return database statement that can delete data
+     */
+    default DbStatement<?, CompletionStage<Long>> createNamedDelete(String statementName, String statement) {
+        return createNamedDmlStatement(statementName, statement);
+    }
+
+    /**
+     * Create andelete statement using a named statement.
+     *
+     * @param statementName the name of the statement
+     * @return database statement that can delete data
+     */
+    DbStatement<?, CompletionStage<Long>> createNamedDelete(String statementName);
+
+    /**
+     * Create a delete statement using a statement text.
+     *
+     * @param statement the statement text
+     * @return database statement that can delete data
+     */
+    DbStatement<?, CompletionStage<Long>> createDelete(String statement);
+
+    /**
+     * Create and execute delete statement using a statement defined in the configuration file.
+     *
+     * @param statementName the name of the configuration node with statement
+     * @param parameters    query parameters to set
+     * @return number of rows deleted from the database
+     */
+    default CompletionStage<Long> namedDelete(String statementName, Object... parameters) {
+        return createNamedDelete(statementName).params(parameters).execute();
+    }
+
+    /**
+     * Create and execute delete statement using a statement passed as an argument.
+     *
+     * @param statement  the delete statement to be executed
+     * @param parameters query parameters to set
+     * @return number of rows deleted from the database
+     */
+    default CompletionStage<Long> delete(String statement, Object... parameters) {
+        return createDelete(statement).params(parameters).execute();
+    }
+
+    /*
+     * DML
+     */
+
+    /**
+     * Create a data modification statement using a named statement passed as an argument.
+     *
+     * @param statementName the name of the statement
+     * @param statement the statement text
+     * @return data modification statement
+     */
+    DbStatement<?, CompletionStage<Long>> createNamedDmlStatement(String statementName, String statement);
```

Comment:
Another occurence of _hybrid_ method with both statement name and statement string.